Tests/SwiftStdlibTests/SequenceExtensionsTests.swift

```diff
@@ -202,4 +202,21 @@ final class SequenceExtensionsTests: XCTestCase {
         let array1 = [Person(name: "Iveta", age: 20, location: Location(city: "Prague"), isStudent: true), Person(name: "Victor", age: 44, location: Location(city: "Dallas"), isStudent: false), Person(name: "Lukasz", age: 62, location: nil), Person(name: "Anna", age: 18, location: Location(city: "Minsk"), isStudent: true)]
         XCTAssertEqual(array1.filter(by: \.isStudent), [Person(name: "Iveta", age: 20, location: Location(city: "Prague"), isStudent: true), Person(name: "Anna", age: 18, location: Location(city: "Minsk"), isStudent: true)])
     }
+
+    func testFirstByKeyPath() {
+        let array1 = [
+            Person(name: "John", age: 30, location: Location(city: "Boston")),
+            Person(name: "Jan", age: 22, location: nil),
+            Person(name: "Roman", age: 26, location: Location(city: "Moscow"))
+        ]
+
+        let first30Age = array1.first(where: \.age, equal: 30)
+
+        XCTAssertNotNil(first30Age)
```

Comment:
This isn't necessary. You'd catch this failed test in the next test.

Reply:
Done.